Release/samples/DSC/dsc_rest_client/dsc_rest_client.cpp

```diff
@@ -0,0 +1,102 @@
+//#define BOOST_ASIO_SAMPLE
```

Comment:
Notice that the first part of the samples is ifdef'ed-out. I'm experimenting with boost::asio